Alignment/APEEstimation/macros/commandsDrawIteration.C

```diff
@@ -18,24 +18,24 @@ gROOT->ProcessLine("DrawIteration drawIteration1(14)");
 drawIteration1.yAxisFixed(true);
 
 //drawIteration1.drawIteration();
-drawIteration1.drawIteration(1,6);
-drawIteration1.drawIteration(7,10);
-drawIteration1.drawIteration(11,22);
-drawIteration1.drawIteration(23,34);
-drawIteration1.drawIteration(35,44);
-drawIteration1.drawIteration(45,64);
+drawIteration1.drawIteration(1,8);
+drawIteration1.drawIteration(9,14);
+drawIteration1.drawIteration(15,26);
+drawIteration1.drawIteration(27,38);
+drawIteration1.drawIteration(39,48);
+drawIteration1.drawIteration(49,68);
 
 
 gROOT->ProcessLine("DrawIteration drawIteration2(15)");
 drawIteration2.yAxisFixed(true);
 
-drawIteration2.drawIteration(1,6);
-drawIteration2.drawIteration(7,10);
-drawIteration2.drawIteration(11,22);
-drawIteration2.drawIteration(23,34);
-drawIteration2.drawIteration(35,44);
-drawIteration2.drawIteration(45,64);
 
+//~ drawIteration2.drawIteration(1,8);
```

Comment:
@mteroerd I realize now that this PR was never signed so I restarted the tests and do another round of review:
why is this commented?

Reply:
This is commented out because usually, we only do the validation plots for iteration 14. I commented out the rest of the code there for consistency.